easybuild/easyconfigs/k/kalign2/kalign2-2.0.4-GCCcore-10.2.0.eb

```diff
@@ -0,0 +1,34 @@
+# Contribution from the NIHR Biomedical Research Centre
+# Guy's and St Thomas' NHS Foundation Trust and King's College London
+# uploaded by J. Sassmannshausen
+
+easyblock = 'ConfigureMake'
+
+name = 'kalign2'
+version = '2.0.4'
+
+homepage = ''
+description = "Kalign is a fast multiple sequence alignment program for biological sequences."
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+
+source_urls = ['http://msa.sbc.su.se/downloads/kalign/']
+sources = ['current.tar.gz']
```

Comment:
best rename this to a download filename and filename with version

Reply:
In light they are don't doing any versioning here, is that a wise thing to do? Else somebody could download a newer version of `current.tar.gz` believes it is the 2.0.4 version and does an `--inject-checksum` and gets into a mess. I don't know what the best practice here is.

Reply:
Yeah this just sucks that they have this awful filename, but if they were to actually update the version, then, at least we won't collide the cache. Anyone doing --inject-checksum is saying that they checked that it is correct and want to change it, so if someone does that incorrectly I have no sympathy there. 

I think we should correct the filename instead of having this "current.tar.gz" file in the local download cache. I don't see how not doing this is in any way an improvement on the situation.

Reply:
Yeah this just sucks that they have this awful filename, but if they were to actually update the version, then, at least we won't collide the cache. Anyone doing --inject-checksum is saying that they checked that it is correct and want to change it, so if someone does that incorrectly I have no sympathy there.

I think we should correct the filename instead of having this "current.tar.gz" file in the local download cache. I don't see how not doing this is in any way an improvement on the situation.

Reply:
That has been addressed. Thanks for your guidance.